Add eval and explain functionalities to UI and APIs (#6)
partiql/partiql-rust-playground: README.md

# PartiQL Playground (Proof of Concept)

> PartiQL Playground provides a functionality to execute PartiQL on client on web environment. It also provides

Reviewer: nit: wording

```suggestion
PartiQL Playground provides the functionality to execute PartiQL in a web environment. It also provides
```
